src/VirtoCommerce.CatalogModule.Data/Model/AssociationEntity.cs

```diff
@@ -104,6 +104,7 @@ public virtual AssociationEntity FromModel(ProductAssociation association)
             Priority = association.Priority;
             AssociationType = association.Type;
             Quantity = association.Quantity;
+            ItemId = association.ItemId;
```

Comment:
Shouldn't we add ItemId to patch aswell?

Reply:
Do we really need it? There is limited set of properties:
            **target.Priority = Priority;
            target.Tags = Tags;
            target.AssociationType = AssociationType;
            target.Quantity = Quantity;**
And even no AssociatedItemId. I think, if I add ItemId, I have to add also AssociatedItemId. Should I?

Reply:
Indeed, as we are searching by ItemId, we could not update it.